docs/table-vaccinations.md

```diff
@@ -47,6 +47,7 @@ counts over time, which is why the cumulative values, if reported, are kept in a
 | Italy | [Commissario straordinario per l'emergenza Covid-19](https://github.com/italia/covid19-opendata-vaccini) | [CC BY](https://github.com/italia/covid19-opendata-vaccini/blob/master/LICENSE.md) |
 | Spain | [Ministry of Health](https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov/vacunaCovid19.htm) | [Attribution required](https://www.mscbs.gob.es/avisoLegal/home.html) |
 | Sweden | [Public Health Agency of Sweden](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/vaccination-mot-covid-19/statistik/statistik-over-registrerade-vaccinationer-covid-19/) | Fair Use |
+| Switzerland | [Federal Office of Public Health](https://www.covid19.admin.ch/en/epidemiologic/vacc-doses?detGeo=CH) | [Copyright, Swiss federal authorities](https://www.admin.ch/gov/en/start/terms-and-conditions.html) |
```

Comment:
Do we need to claim "Fair Use" here as is done for Sweden?

Reply:
I don't know actually. For Swiss I found https://www.admin.ch/gov/en/start/terms-and-conditions.html which says "Copyright, Swiss federal authorities" so I thought that would be enough

Reply:
I would set the text as "Fair Use" and keep the link to the terms. We'll consult with our legal counsel but let's not block on it.

Reply:
done